enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/app-embed-sdk.tsx

```diff
@@ -0,0 +1,27 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+
+import { SdkIframeEmbedRoute } from "./components/SdkIframeEmbedRoute";
+
+function _init() {
+  document.body.style.margin = "0";
+  document.body.style.backgroundColor = "transparent";
+
+  const rootElement = document.getElementById("root");
+
+  if (!rootElement) {
```

Comment:
I wish there's a little bit of console log, or a comment for us just to know this shouldn't happen. i.e. we control how the HTML is served and this case shouldn't really happen. It's just that TypeScript being whiny and we need to satisfy it 😆

Reply:
good point! let me add a console log

Reply:
Fixed in https://github.com/metabase/metabase/pull/56915/commits/bd189b7862dbba16903cc67643712c086603b74c